src/components/info-list.jsx

```diff
@@ -2,17 +2,18 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 import '../styles/components/info-list.scss';
-
+// NOTE: DELETE? is this used???
```

Comment:
Can this be tested and if it is not used remove from the code?

Reply:
That was a mistake, it is still used.